### PR TITLE
Removed non-existant win64 option for os.platform

### PR DESCRIPTION
**Pull request by cchamberlain**
Per https://nodejs.org/api/os.html#os_os_platform, `process.env.platform` and `require('os').platform()` are equivalent.  Although the name is confusing, `win32` refers to all Windows architectures and can be reliably used to detect Windows.

### CODE REVIEW

**Comment by shapeshed** [comment]
Thanks for this. I think this must have changed a while back. It is hard to parse the Changelogs to find out when this happened. Anyway thanks!

**Comment by cchamberlain** [comment]
I've definitely seen this mentioned a handful of places so I've no doubt of that. Kudos on the pull request update structure!
